### ISSUE TITLE
Incompatibility with dask 2024.11.2

### ISSUE THREAD

**Issue by GaetanLepage** [itamarst/eliot#505]
Two tests suite currently fails with the latest version of `dask` (2024.11.2):
```
=========================== short test summary info ============================
FAILED eliot/tests/test_dask.py::DaskTests::test_persist_pandas - AttributeError: 'HighLevelGraph' object has no attribute 'clear'
FAILED eliot/tests/test_dask.py::AddLoggingTests::test_add_logging_to_full_graph - AttributeError: 'HighLevelGraph' object has no attribute 'clear'
========== 2 failed, 479 passed, 16 skipped, 5086 warnings in 50.74s ===========
```

**Comment by itamarst** [comment]
Thanks for the bug report, I'll take a look.

**Comment by itamarst** [comment]
I just released 1.16.0 which should fix it. Let me know if you encounter any other issues.

**Comment by GaetanLepage** [comment]
Indeed, it works great. Thanks for the quick fix !
